recipe_scrapers/tasty.py

```diff
@@ -0,0 +1,28 @@
+from ._abstract import AbstractScraper
+
+
+class Tasty(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "tasty.co"
+
+    def title(self):
+        return self.schema.title().replace(" Recipe by tasty", "")
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        return self.schema.instructions().replace("\n", "")
```

Comment:
Upon further review, I believe that this creates an inconsistency with the rest of the package. If you get the instructions for the recipe shown in the usage example it contains newlines.

It returns as follows:
`'Preheat an outdoor grill for medium-high heat and lightly oil grate.\nWhile the grill is preheating, mix together eggs, garlic, feta cheese, spinach, and turkey in a large bowl until well combined; form into 8 patties.\nCook on preheated grill until no longer pink in the center, 15 to 20 minutes.'`

Reply:
totally agree